assets/components/payPalExpressButton/payPalExpressButton.jsx

```diff
@@ -53,4 +41,4 @@ function mapDispatchToProps(dispatch) {
 
 // ----- Exports ----- //
 
-export default connect(mapStateToProps, mapDispatchToProps)(PayPalExpressButton);
+export default connect(undefined, mapDispatchToProps)(PayPalExpressButton);
```

Comment:
`null` is a value and `undefined` is the absence of value. In order to trigger the default parameter behaviour, we should use `undefined`. More info [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)